src/promptflow-devkit/promptflow/_cli/_params.py

```diff
@@ -94,8 +114,9 @@ def add_param_set_positional(parser):
 
 def add_param_environment_variables(parser):
     parser.add_argument(
+        "-env",
```

Comment:
we don't need to support -env , --env is ok

Reply:
change to --env